indexer-app/resources/schemas/dataqualitysummary/v1.0.0/schema.json

```diff
@@ -5,6 +5,10 @@
   "additionalProperties": false,
   "type": "object",
   "properties": {
+    "Id": {
+      "description": "The identifier that was assigned to this record by an older system. It is in the schema to preserve search API functionality in order to store this identifier.",
+      "type": "string"
+    },
     "Name": {
       "description": "The name associated with the quality summary of the data set.",
       "type": "string"
```

Comment:
I'm a confused about this file. Is it the schema file for the dataqualitysummary metadata? If so, why is it under indexer?

Reply:
Yes, it is. There is no difference.  Right now we put both the index.json and schema.json file in both the indexer-app and in ingest-app.  I want to put these into 1 location, but we need to figure out how to put these resources in each micro service (ingest, indexer) just in case a developer just wants to compile and run 1 of them without the other services.  I haven't had time to do that yet.